src/main/scala/viper/gobra/ast/internal/Program.scala

```diff
@@ -1063,6 +1063,11 @@ case class ShiftRight(left: Expr, right: Expr)(val info: Source.Parser.Info) ext
 }
 case class BitNeg(op: Expr)(val info: Source.Parser.Info) extends IntOperation
 
+/*
+ * Convert 'expr' to non-interface type 'newType'. If 'newType' is
+ * an interface type, then 'ToInterface' should be used instead.
```

Comment:
Could we add an assertion checking that `newType` is not an interface?

Reply:
To do that, we would need to either be able to compute the underlyingType in this class (which we cannot because we do not have typing information) or to pass the underlyingType as a parameter (which is cumbersome and may not perform as intended if one forgets to compute the actual underlyingType).

Reply:
I see thanks